components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/dao/LocalClaimDAO.java

```diff
@@ -221,6 +221,51 @@ public void updateLocalClaim(LocalClaim localClaim, int tenantId) throws ClaimMe
         }
     }
 
+    /**
+     * Update attribute claim mappings related to tenant id and domain.
+     *
+     * @param localClaimList  list of local claims.
```

Comment:
Fix case.

Reply:
Addressed with commit 2e3f3c951772d195bcbedb1a7b81f4815b531259 in https://github.com/wso2/carbon-identity-framework/pull/3774 PR